src/h5web/explorer/Icon.tsx

```diff
@@ -34,7 +33,7 @@ function Icon(props: Props) {
     );
   }
 
-  const LeafIcon = LEAF_ICONS[entity.kind];
+  const LeafIcon = entity.kind ? LEAF_ICONS[entity.kind] : FiLink;
```

Comment:
The link icons is now used as a fallback, when an entity doesn't have a `kind` (i.e. when it's unresolved).